packages/sdk/src/next/NextInitiator.ts

```diff
@@ -0,0 +1,28 @@
+import { Connection } from 'penpal';
+import { ConfigType, EditorAPI } from '../types/CommonTypes';
+import { SubscriberController } from './controllers/SubscriberController';
+
+export class NextInitiator {
```

Comment:
Perhaps this is a JS thing but I have no idea what this class name even means. Also 'next' is confusing

Reply:
I'll add documentation which will make the intentions more clear

Reply:
'Initiator' what is it 'initiating' 🤔

Reply:
next features, other names are welcome but I don't really know, since it's not really a controller or is it?

Reply:
`NextIncubator` 💯 _where the **next** set of **breaking changes** come to **hatch**_ 😆